### PR TITLE
phpbench: init at 1.2.10

### PR DESCRIPTION
**Pull request by drupol**
PHPBench: https://github.com/phpbench/phpbench

CC: @dantleech

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/misc/phpbench/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, makeBinaryWrapper, lib, php80 }:
```
Comment (pstn) [inline pkgs/development/tools/misc/phpbench/default.nix:1]: Is there a reason to use `php80` over just `php`?

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -17232,6 +17232,8 @@ with pkgs;
 
   ansible-doctor = with python3.pkgs; toPythonApplication ansible-doctor;
 
+  phpbench = callPackage ../development/tools/misc/phpbench { };
```
Comment (pstn) [inline pkgs/top-level/all-packages.nix:17235]: If the php version needs to be overwritten, I think the better way is to do it here with `{ php = php80; }`

File: pkgs/development/tools/misc/phpbench/default.nix (hunk 1 above)
Comment (drupol) [reply pkgs/development/tools/misc/phpbench/default.nix:1]: `php81`  is throwing deprecations.

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (drupol) [reply pkgs/top-level/all-packages.nix:17235]: Ok will do the change.

File: pkgs/development/tools/misc/phpbench/default.nix (hunk 1 above)
Comment (dantleech) [reply pkgs/development/tools/misc/phpbench/default.nix]: for phpbench? it shouldn't be...

Comment (drupol) [reply pkgs/development/tools/misc/phpbench/default.nix]: I'll copy paste a screenshot tomorrow morning ;)

**Comment by drupol** [comment]
Closing this PR since it has been included in #225401

**Pull request closed without merging**